packages/serverless-collector/src/activate.js

```diff
@@ -15,10 +15,11 @@ const { normalizeConfig } = coreUtil;
 let logger = consoleLogger;
 
 const config = normalizeConfig({});
+config.logger = logger;
 
 function init() {
   if (process.env.INSTANA_DEBUG || process.env.INSTANA_LOG_LEVEL) {
-    logger.setLevel(process.env.INSTANA_DEBUG ? 'debug' : process.env.INSTANA_LOG_LEVEL);
+    logger.setLevel(process.env.INSTANA_DEBUG === 'true' ? 'debug' : process.env.INSTANA_LOG_LEVEL);
```

Comment:
We need to do the string boolean check in line 21.
Otherwise you can call `setLevel(null)` if INSTANA_DEBUG is not 'true'.